.terraform.lock.hcl

```diff
@@ -0,0 +1,21 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
Is this file meant to be checked in?

Reply:
Looks that way, going by the [official documentation](https://www.terraform.io/upgrade-guides/0-14.html#provider-dependency-lock-file):

> We strongly recommend that you commit this file to your version control system